Task_DEV_4/Task_DEV_4/Parallelogram.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
```

Comment:
It would be better to have there one blank line between usings and namespace.

Reply:
Done.